wavesrv/pkg/remote/remote.go

```diff
@@ -2064,8 +2073,6 @@ func (msh *MShellProc) handleCmdDonePacket(donePk *packet.CmdDonePacketType) {
 			// fall-through (nothing to do)
 		}
 	}
-
-	go pushNumRunningCmdsUpdate(&donePk.CK, -1)
```

Comment:
moved this functionality into UpdateCmdDoneInfo, which gets called above in this function

is this okay?